code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -235,3 +250,27 @@
 	strip_delay = 130
 	max_heat_protection_temperature = FIRE_IMMUNITY_HELM_MAX_TEMP_PROTECT
 	unacidable = 1
+
+//Carpsuit, bestsuit, lovesuit
+/obj/item/clothing/head/helmet/space/hardsuit/carp
+	name = "carp helmet"
+	desc = "Spaceworthy and it looks like a space carp's head, smells like one too."
+	icon_state = "carp_helm"
+	item_state = "syndicate"
+	armor = list(melee = -20, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 100, rad = 75)	//As whimpy as a space carp

```

Comment:
Someone told me in IRC it would, I'll test it now.

I was so over the moon about getting a hooded thing to work that I forgot to even try.